### DISCUSSION TITLE
Templater and Database Folder plugin

### DISCUSSION THREAD

**Discussion by jim-hart-dev** [SilentVoid13/Templater#675] [Help] [answered]
I'm trying to get Templater to work with the Database Folder plugin when creating new rows in the data table. I believe the issue is that Database Folder adds the frontmatter content before Templater can fire and therefore the file size is > 0. As a test, I modified the main.js and removed the `file.stat.size == 0` check and it worked. Is there a better solution?

**Comment by AB1908** [comment, marked as answer]
Seems to be an order of operations thing. I'd suggest reaching out to the dev of the other plugin and asking them if they could add an option/logic to handle templater stuff.

**Reply by jim-hart-dev** [reply]
That could work. What about an option in Templater to ignore frontmatter when checking for existing file content?

**Reply by AB1908** [reply]
Hmm you'd have to script that out yourself. You can execute JavaScript and access the obsidian API in `<%*` blocks.
